redisinsight/ui/src/telemetry/telemetryUtils.ts

```diff
@@ -19,6 +20,8 @@ import {
 import { TelemetryEvent } from './events'
 import { checkIsAnalyticsGranted } from './checkAnalytics'
 
+export const getProvider = (): string | undefined => get(store.getState(), 'connections.instances.connectedInstance.provider', undefined)
```

Comment:
instead of `T | undefined`, or `T | null` we use `Maybe<T>` and `Nullable<T>`

Reply:
`get(store.getState(), 'connections.instances.connectedInstance.provider')` (without 3d argument) will work the same, no?